docs/en/advanced-concepts/migrating-to-appium-2.0.md

```diff
@@ -32,6 +32,26 @@ npm install -g appium --drivers=xcuitest,uiautomator2
 
 This will install Appium and the two drivers for you in one go.
 
+### :bangbang: Chromedriver installation flags
+
+In Appium 1.x it was possible to customize the way Chromedriver was installed (as part of the UiAutomator2 driver for example), using the following command line flags:
+
+* `--chromedriver-skip-install`
+* `--chromedriver-version`
+* `--chromedriver-cdnurl`
+
+Because Appium 2.0 now installs drivers for you, and because these flags were implemented as NPM config flags, they will no longer work. Instead, use the following environment variables during driver installation:
+
+* `APPIUM_CHROMEDRIVER_SKIP_INSTALL`
```

Comment:
`APPIUM_SKIP_CHROMEDRIVER_INSTALL`? https://github.com/appium/appium-chromedriver#skipping-binary-installation

Reply:
oh yeah you're right